pkg/blueprint/blueprint_test.go

```diff
@@ -23,16 +23,16 @@ version = "2.4.*"
 
 [[customizations.filesystem]]
 mountpoint = "/var"
-size = 2147483648
+minsize = 2147483648
 
 [[customizations.filesystem]]
 mountpoint = "/opt"
-size = "20 GB"
+minsize = "20 GB"
```

Comment:
Maybe we can also organize the test a little bit differently so that there is less risk of "drift" between json/toml parsing? I.e. make the blueprint in json/toml the same and after parsing compare that the two are identical.

I attached 
[0001-blueprint-extend-TestBlueprintParse-to-make-drift-le.patch.txt](https://github.com/user-attachments/files/16834790/0001-blueprint-extend-TestBlueprintParse-to-make-drift-le.patch.txt) with the idea.